eclair-core/src/main/scala/fr/acinq/eclair/payment/relay/ChannelRelayer.scala

```diff
@@ -150,7 +150,7 @@ object ChannelRelayer {
       case Some(nextNodeId) =>
         log.debug(s"next hop for htlc #{} is nodeId={}", add.id, nextNodeId)
         // then we retrieve all known channels to this node
-        val allChannels = node2channels.getOrElse(nextNodeId, Set.empty[ShortChannelId])
+        val allChannels = node2channels.get(nextNodeId)
```

Comment:
Isn't that a regression if `nextNodeId` isn't found?

Reply:
Note that `node2channels` is now a `MultiDicts`. Here is the signature of this method:

```scala
  def get(key: K): Set[V] = sets.get(key).getOrElse(Set.empty)
```